### PR TITLE
Install global font family

### PR DESCRIPTION
**Pull request by herecomesjaycee**
This is the basis of importing lato font family to the app globally. Feel free to merge if any one is happy with this approach.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,6 +2,9 @@
   text-align: center;
 }
 
+* {
+  font-family: lato;
```
Comment (GenesisSelf) [inline src/App.css:6]: Is this font safe for all browsers?

Comment (GenesisSelf) [reply src/App.css:6]: I just wonder if there should be a backup font that can take over incase `lato` doesn't render properly.

Comment (herecomesjaycee) [reply src/App.css:6]: https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/

We can ask @msajara  which font would she prefer for fallback?

Comment (herecomesjaycee) [reply src/App.css:6]: @GenesisSelf if this is non-blocking, we can always merge so people can use it then come back to add fallback font

**Review by jopfre — COMMENTED** [review]
I think it should be a regular dependency in package.json, not a dev dependency no?

**Comment by herecomesjaycee** [comment]
I've followed the installation [instruction](https://www.npmjs.com/package/lato-font), not entirely sure why they recommended installing in dev dependency, other than webpack would do the heavy lifting during compilation.

**Comment by herecomesjaycee** [comment]
@jopfre Merging in now and will test in production if it would cause any hiccups.

**Pull request merged**